### DISCUSSION TITLE
issue

### DISCUSSION THREAD

**Discussion by arunprabu563** [Cyfrin/foundry-full-course-cu#1263] [Q&A]
FundME fundMe = new FundME();
unused local variable error , dont know what to do

**Comment by xyz899** [comment]
## Hi,

### This pops out when you don't use avariable but it's supposed to be a warning not an error meaning you can still compile and run.
**Use fundMe in your file in order to remove it for example :**

```
// SPDX-License-Identifier: UNLICENSED
pragma solidity ^0.8.13;

import "forge-std/Test.sol";
import 'src/HelloWorld.sol';

contract HelloWorldTest is Test {
   HelloWorld hellofund;

   function setUp() public {
       hellofund = new HelloWorld();
   }

   function test1() public {
       assertEq(
           hellofund.greet(),
           "John Marston"
       );
   }
}
```

**Reply by arunprabu563** [reply]
It's a warning only , I did everything for it to go, did exactly in the tutorial but it never went

**Reply by arunprabu563** [reply]
![IMG_20231230_141434.jpg](https://github.com/Cyfrin/foundry-full-course-f23/assets/85756382/172ad524-b03a-453a-8f45-286edf9111d2)

**Reply by xyz899** [reply]
Like I said It is because you don't use it refer to the example where I used a contract with a greet function in it. `hellofund.greet()` for example then the warning will be removed. Solidity by default tells you what vars are not used so you can gas optimize your contract

**Reply by xyz899** [reply]
like I said a warning won't do anything but take more storage for the variable so ignore it for now I guess or use it, refer to my answer
